spec/support/factory_girl.rb

```diff
@@ -7,4 +7,28 @@
     password_confirmation 'password'
   end
 
+  factory :tour do
+    name 'San Francisco Bars'
+    user_id 1
+
+    user
+  end
+
+  factory :stop do
+    name 'Zeitgeist'
+    description 'Kick ass outdoor bar, great selection of $4.50 beers.'
+    latitude 37.770005
+    longitude -122.422120
+    stop_length 120
+  end
+
+  factory :tour_stop do
+    tour_id 1
+    stop_id 1
+    stop_number 1
+
+    tour
+    stop
+  end
```

Comment:
When you declare `tour` and `stop` in a factory, you are relying on the factory to create a record and set the id of that record for you. Whenever you think you need an `_id` in a rails app, stop yourself. There is probably a better way.

Delete `tour_id` and `stop_id`, and any other instance of `_id` in this file.

You could [sequence](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences) the stop number